PhysIO/code/readin/tapas_physio_read_physlogfiles.m

```diff
@@ -93,6 +93,9 @@
     case 'siemens_hcp'
         [c, r, t, cpulse, acq_codes, verbose] = ...
             tapas_physio_read_physlogfiles_siemens_hcp(log_files, cardiac_modality, verbose);
+    otherwise
+        error('Invalid vendor "%s". See tapas_physio_new for supported vendors',...
```

Comment:
Please use `tapas_physio_log` here instead of `error`, in order to write the error message back to `physio.verbose`

e.g.,

See example, e.g., in `tapas_physio_get_cardiac_pulses_auto_matched`, l 78 (2 is error level, 1 is warning, 0 is info)

```
  stringError = sprintf(['Not enough pulse events (%d<%d) in physiological time series to perform backwards-search ' ...
            'for first pulse wave peak. Please check peak thresholding, data quality or total length of time series.'], ...
            nPeaksTotal, nPeaksRequired);
            verbose = tapas_physio_log(stringError, verbose, 2);
```

Reply:
Done